dhis-2/dhis-e2e-test/docker-compose.e2e.yml

```diff
@@ -1,9 +1,12 @@
 version: "3"
```

Comment:
Version here is different from the one in the other docker compose file. Is it a problem?

Reply:
Will adjust it so it is the same as in the other file 👍🏻